nfl_data_py/__init__.py

```diff
@@ -228,14 +260,22 @@ def import_weekly_data(years, columns=None, downcast=True):
     if min(years) < 1999:
         raise ValueError('Data not available before 1999.')
     
-    if columns is None:
+    if not columns:
         columns = []
         
     # read weekly data
     url = r'https://github.com/nflverse/nflverse-data/releases/download/player_stats/player_stats_{0}.parquet'
     data = pandas.concat([pandas.read_parquet(url.format(x), engine='auto') for x in years])
 
-    if len(columns) > 0:
+    if thread_requests:
```

Comment:
Skipped over a little logic change here. As is, it always make the original sequential requests, then optionally requests them _again_ if `thread_requests=True`.

It should be something like 
```
columns = columns or None
if thread_requests:
    with ThreadPoolExecutor() as executor:
        futures = [executor.submit(_read_parquet, url.format(year), year, columns) for year in years]
        data = [future.result() for future in as_completed(futures)]
        data = sorted(data, key=lambda x: x['year'])
        data = [x['data'] for x in data]
else:
    data = [pandas.read_parquet(url.format(x), columns=columns) for x in years]
```

(bonus refactoring to get rid of the columns post-filtering just below this)